src/Symfony/Component/DomCrawler/Crawler.php

```diff
@@ -134,11 +132,8 @@ public function add($node)
      * If the charset is not set via the content type, it is assumed to be UTF-8,
      * or ISO-8859-1 as a fallback, which is the default charset defined by the
      * HTTP 1.1 specification.
-     *
-     * @param string      $content A string to parse as HTML/XML
-     * @param string|null $type    The content type of the string
      */
-    public function addContent($content, $type = null)
+    public function addContent(string $content, string $type = null)
```

Comment:
If the original PHPdoc was `string|null`, the type-hint should be `?string`, right?

Reply:
we don't add the `?` when we have `= null` making it nullable already.